### ISSUE TITLE
accessibility verifyed chats are not recognisable for blind people

### ISSUE THREAD

**Issue by amirmahdifard** [UnigramDev/Unigram#2989]
so, even in the last version of unigram, this problem exists, and i never saw any commit fixing this issue: when you are walking throo your chat list with your arrow keys, verifyed channels or bots or groupes are not accessible: it's even not accessible when you are inside the chat or inside the profile: there are some other accessibility problems too that i've listed them for you pending for fix, but apparently telling you to fix accessibility in unigram insiders groupe doesn't work, opening tikits in github doesn't work, listing them in a telegram channel and giving them to you doesn't work, as i tryed all these ways and now it's for a long time that you haven't fixed any accessibility bugs as i see the commits and stuf: i don't know what to do next.

**Comment by MarcAnt01** [comment]
Hi, bugs reports are required to follow the issue template. When creating a new issue select "Bug report" and fill all the required information

**Comment by amirmahdifard** [comment]
@MarcAnt01 i'm sorry, but you don't make any sence: if it was a new bug, i would do it, but this bug is an old bug that i told the developer at least 100 times: you are just trying to find an advise to close my tikits: onnistly i'm done with it.

**Comment by MarcAnt01** [comment]
@amirmahdifard if this is not a new issue, then just wait for the fix. You are constantly violating our code of conduct. This is your last warning, if you continue to behave in an uncivilized way you'll be blocked from the GitHub repository.

**Comment by amirmahdifard** [comment]
@MarcAnt01 i suggest you look at this channel that the developer is even joind on it, and see the times of my postes and look at my list before saying such things:
https://t.me/+3wJGL85diGxjZjk8
thanks.

**Comment by amirmahdifard** [comment]
@MarcAnt01 at least i'm happy i didn't do this every day or something, it's now a pretty long time that there are no accessibility fix.

**Comment by MarcAnt01** [comment]
@amirmahdifard please stop, you have no right to insist on a FOSS project to get your fixes done.

**Comment by amirmahdifard** [comment]
even if they haven't fixed for a long time or even never? ok, i'll stop, but i thort that issues are there and they are submitting to be fixed and to get done at some points: but apparently it's not true about this project, so i'm sorry for the mis understandings that i had, and i'll stop.
